index.js

```diff
@@ -16,6 +16,19 @@ const createAndTagTable = async function createAndTagTable(region, tableName, se
     region,
   })
   try {
+
+    try {
```

Comment:
i think this check should be moved before the first consoleLog (line 13)

Reply:
ah i missed that log line. it's kind of redundant now that this check logs if it does or does not create the table.